Mathlib/Algebra/Module/Torsion/Field.lean

```diff
@@ -0,0 +1,24 @@
+/-
+Copyright (c) 2025 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Field.Defs
+import Mathlib.Algebra.GroupWithZero.Action.Units
+import Mathlib.Algebra.Module.Torsion.Free
+
+/-!
+# Vector spaces are torsion-free
+
+In this file, we show that any module over a division semiring is torsion-free.
+
+Note that more generally any reflexive module is torsion-free.
+-/
+
+open Module
+
+variable {𝕜 M : Type*} [DivisionSemiring 𝕜] [AddCommMonoid M] [Module 𝕜 M]
+
+/-- Any (semi)vector space is torsion-free. -/
+instance Semifield.to_moduleIsTorsionFree : IsTorsionFree 𝕜 M where
```

Comment:
Shouldn't this be more like `Module.instIsTorsionFreeOfDivisionSemiring`? Or better yet, just let Lean name it:
```suggestion
instance : IsTorsionFree 𝕜 M where
```